modules/nw-operator-cr.adoc

```diff
@@ -346,6 +346,10 @@ The default value is `false`.
 This field has an interaction with the Open vSwitch hardware offloading feature.
 If you set this field to `true`, you do not receive the performance benefits of the offloading because egress traffic is processed by the host networking stack.
 
+|`ipForwarding`
+|`object`
+|You can control IP forwarding for all traffic on OVN-Kubernetes managed interfaces by using the `ipForwarding` specification in the `Network` resource. Specify `Restricted` to only allow IP forwarding for Kubernetes related traffic. Specify `Global` to allow forwarding of all IP traffic. For new installations, the default is `Restricted`. For upgrades to {product-title} 4.14, the default is `Global`.
```

Comment:
[Update is the preferred term](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update) over upgrade:

```suggestion
|You can control IP forwarding for all traffic on OVN-Kubernetes managed interfaces by using the `ipForwarding` specification in the `Network` resource. Specify `Restricted` to only allow IP forwarding for Kubernetes related traffic. Specify `Global` to allow forwarding of all IP traffic. For new installations, the default is `Restricted`. For updates to {product-title} 4.14, the default is `Global`.
```